service/controller/key/app.go

```diff
@@ -41,14 +41,10 @@ func AppCatalogSecretNamespace(customResource v1alpha1.AppCatalog) string {
 	return customResource.Spec.Config.Secret.Namespace
 }
 
-// AppConfigMapName returns the name of the configmap that stores app level
-// config for the provided app CR.
```

Comment:
Some of the key funcs have godoc and others don't. I'd like this to be consistent.

The names are clear so I think we should just drop the godoc.